libs/nspr/patches/001-Makefile.in_rm_BUILD_STRING_and_BUILD_TIME.patch

```diff
@@ -24,7 +24,7 @@ Signed-off-by: Mingli Yu <Mingli.Yu@windriver.com>
 
 --- a/nspr/lib/ds/Makefile.in
 +++ b/nspr/lib/ds/Makefile.in
-@@ -110,13 +110,7 @@ GARBAGE += $(TINC)
+@@ -104,13 +104,7 @@ GARBAGE += $(TINC)
```

Comment:
I found that Yocto has this patch, but Debian does not.

Could you please verify if this patch is still required?